packages/ws/src/utils/constants.ts

```diff
@@ -20,6 +20,7 @@ export enum Encoding {
 export enum CompressionMethod {
```

Comment:
Just out of curiousity, could you do

```ts
export enum CompressionMethod {
	ZlibNative = 'zlib-stream',
	ZlibSync = 'zlib-stream',
	ZstdNative = 'zstd-stream',
}
```

and just remove the `CompressionParameterMap` or was there a functional reason not to use it?

Reply:
We used to do this, but I changed it in #10243; memory is doing me dirty right now so I wouldn't be able to tell you why, but I definitely remember the enum values were terribly annoying for some reason.

Reply:
Got it. It's because of the switch statements where we do `case CompressionMethod.ZlibNative` and such. We need to actually distinguish between ALL of the types, which an enum of that sort would make impossible